### PR TITLE
add presto in docker example

### PR DESCRIPTION
**Pull request by mkorolyov**
this example runs jaeger server as data producer, scylladb, presto and metabase. All in single docker compose.

### CODE REVIEW

File: presto/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Instructions
+============
+
+**Procedure**
+1. Run `docker-compose up -d` to start the containers
+2. Wait for about minute for ScyllaDB to start and Metabase to initialize.
+3. Open Jaeger HotRod sample application at http://localhost:8081 and click on buttons to generate traces.
+4. Open Metabase at http://localhost:3000 and complete admin onboarding.
+5. Add Presto as a data source in Metabase. 
+Use `http://presto:8080` as a host and `scylladb` as a catalog name.
```
Comment (guy9) [inline presto/README.md:21]: change http://presto -> presto

File: presto/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+**Procedure**
+1. Run `docker-compose up -d` to start the containers
+2. Wait for about minute for ScyllaDB to start and Metabase to initialize.
+3. Open Jaeger HotRod sample application at http://localhost:8081 and click on buttons to generate traces.
+4. Open Metabase at http://localhost:3000 and complete admin onboarding.
+5. Add Presto as a data source in Metabase. 
+Use `http://presto:8080` as a host and `scylladb` as a catalog name. 
+During filling presto connection details fill `Username` with arbitrary string 
+and leave user `Password` empty.
+6. Observe traces insights in Metabase.
```
Comment (guy9) [inline presto/README.md:24]: How? Elaborate on this part.

**Review by guy9 — CHANGES_REQUESTED** [review]

File: presto/README.md (hunk 1 above)
Comment (mkorolyov) [reply presto/README.md]: thanks, fixed

File: presto/README.md (hunk 2 above)
Comment (mkorolyov) [reply presto/README.md]: Added example steps

**Pull request merged**